### PR TITLE
adding jupyter notebook with voting classifier example

### PR DESCRIPTION
**Pull request by dvatterott**
Pull request for issue #252 (https://github.com/dask/dask-ml/issues/252) regarding adding a voting classifier example.

### CODE REVIEW

**Comment by mrocklin** [comment]
cc @TomAugspurger

File: machine-learning/voting-classifier.ipynb [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Voting Classifier\n",
+    "\n",
+    "A [Voting classifier](http://scikit-learn.org/stable/modules/ensemble.html#voting-classifier) model combines multiple different models into a single model, which is (ideally) stronger than any of the individual models alone. \n",
+    "\n",
+    "Dask enables users to train more individual models in parallel than would have been otherwise possible, which prevents the additional classifier models from needing additional training time.\n",
```
[the remark this answers is not in the document]
Comment (TomAugspurger) [reply machine-learning/voting-classifier.ipynb]: It'd also be good to link to the Dask-ML docs on distributed joblib: http://dask-ml.readthedocs.io/en/latest/joblib.html

File: machine-learning/voting-classifier.ipynb [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+   "metadata": {},
+   "source": [
+    "I see no advantage of using dask because sklearn is already using all my computers processors, but dask enables users to take advantage of cluster computing and train models across multiple different machines."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Record software used in this notebook."
```
Comment (TomAugspurger) [inline machine-learning/voting-classifier.ipynb]: I have mixed thoughts about this. It's nice from a reproducibility point of view, but I think it's a bit distracting from the message. I think curious users will be able to find this repo, with the exact binder image and all that.

**Comment by dvatterott** [comment]
@TomAugspurger. Thanks for the comments. I'll try to carve out time tonight to make these edits and push up another commit.

**Comment by dvatterott** [comment]
Good call. Thanks. Change made.

**Comment by TomAugspurger** [comment]
Thanks @dvatterott!

**Pull request merged**